lib/app.js

```diff
@@ -30,6 +32,8 @@ module.exports = function App(resolvers) {
     mongodb: new Resolver(new MongoDBSource()),
     redis: new Resolver(new RedisSource()),
     yarn: new Resolver(new YarnSource()),
+    "php-scalingo-18": new Resolver(new PhpScalingo18Source()),
+    "php-scalingo-14": new Resolver(new PhpScalingo14Source()),
     "php-scalingo": new Resolver(new PhpScalingoSource()),
```

Comment:
`php-scalingo` and `php-scalingo-14` are identical, isn't it? Can't we factorize the code so that it's not a copy-paste?

Reply:
You're right, done